lib/src/executables/null_safety_migrator_companion.dart

```diff
@@ -39,13 +39,37 @@ void main(List<String> args) async {
   final parsedArgs = parser.parse(args);
   final dartPaths = allDartPathsExceptHiddenAndGenerated();
 
-  exitCode = await runInteractiveCodemod(
+  exitCode = await runInteractiveCodemodSequence(
```

Comment:
I separated these out because I noticed when testing in web_skin_dart that the patches were messing eachother up sometimes

Reply:
Hm, did you try with `runInteractiveCodemodSequence` without `aggregate`? 
```dart
runInteractiveCodemodSequence(dartPaths
  dartPaths, 
  [
    CallbackRefHintSuggestor(),
    ClassComponentRequiredInitialStateMigrator(),
    StateMixinSuggestor()
  ]
```

runInteractiveCodemodSequence _should_ behave the same as back-to-back runInteractiveCodemod/runInteractiveCodemodSequence calls.

Reply:
Similar thread in the other PR, probably a package:codemod bug: https://github.com/Workiva/over_react_codemod/pull/297#discussion_r1803856176